umbra-js/src/classes/Umbra.ts

```diff
@@ -381,9 +381,7 @@ export class Umbra {
     const endBlock = overrides.endBlock || 'latest';
 
     // Try querying events using Goldsky, fallback to querying logs.
-    // Goldsky fetching uses the browser's `fetch` method to query the subgraph, so we check
-    // that window is defined first to avoid trying to use fetch in node environments
-    if (typeof window !== 'undefined' && this.chainConfig.subgraphUrl) {
+    if (this.chainConfig.subgraphUrl) {
```

Comment:
Looks like CI is failing, haven't looked closely to see if it's related, but code wise the change looks right

Reply:
Hey @garyghayrat just bumping this

Reply:
Think the local subgraph being mistakenly set to polygon one was causing the CI problems. https://github.com/ScopeLift/umbra-protocol/pull/591/commits/a4df87bd466edad4ec03c0a65c32a07d6f171dcc should fix it.